openhtf/core/measurements.py

```diff
@@ -139,6 +140,19 @@ def _initialize_value(self):
     else:
       self.measured_value = MeasuredValue(self.name)
 
+  def __deepcopy__(self, memo):
```

Comment:
Ah makes sense since the purpose of _initialize_value is just to set measure_valued